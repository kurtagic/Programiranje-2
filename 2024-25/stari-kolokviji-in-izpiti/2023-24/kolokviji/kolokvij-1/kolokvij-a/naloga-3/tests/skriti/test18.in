75 4
629 0 0 0
0 0 0 0
0 0 0 -932
0 552 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 844 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 196 0
0 0 0 0
0 -581 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 -845
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 -239
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 -764
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 547
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
371 0 0 0
0 0 0 0
0 0 -627 0
0 0 0 0
-284 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
0 0 0 0
-630 0 0 0
0 0 0 0
0 0 288 0
0 0 0 0
0 0 0 -306
0 0 0 0
0 0 0 0
0 0 0 0
756 0 0 0
