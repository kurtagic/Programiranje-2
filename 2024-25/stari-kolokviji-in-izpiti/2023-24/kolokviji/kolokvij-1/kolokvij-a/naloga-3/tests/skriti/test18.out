629_-932_552_844_196_-581_-845_-239_-764_547_371_-627_-284_-630_288_-306_756
0_1_1_2_3_3_3_3_3_4_4_4_4_4_4_4_4_5_5_6_6_6_6_6_6_6_6_6_6_6_7_7_7_7_8_8_8_8_8_9_9_9_9_9_9_9_10_10_10_10_10_10_10_10_10_10_10_11_11_12_12_13_13_13_13_13_13_14_14_15_15_16_16_16_16_17
0_3_1_2_2_1_3_3_3_3_0_2_0_0_2_3_0
